browser/brave_wallet/brave_wallet_service_delegate_helper.cc

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/brave_wallet_service_delegate.h"
+
+#include <memory>
+
+#include "build/build_config.h"
+#include "brave/browser/brave_wallet/brave_wallet_service_delegate_impl.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_service_delegate.h"
+#include "content/public/browser/browser_context.h"
+
+namespace brave_wallet {
+
+// static
+std::unique_ptr<BraveWalletServiceDelegate> BraveWalletServiceDelegate::Create(
```

Comment:
why can't we put this implementation in `brave_wallet_service_delegate_impl.cc`?
I don't understand the need of helper.cc while that static function belongs to `BraveWalletServiceDelegate`

Reply:
because it isn't included in android